invokeai/frontend/web/src/features/nodes/components/sidePanel/WorkflowListMenu/SaveWorkflowButton.tsx

```diff
@@ -0,0 +1,46 @@
+import { IconButton } from '@invoke-ai/ui-library';
+import { $builtWorkflow } from 'features/nodes/hooks/useWorkflowWatcher';
+import { useSaveWorkflowAsDialog } from 'features/workflowLibrary/components/SaveWorkflowAsDialog/useSaveWorkflowAsDialog';
+import { isWorkflowWithID, useSaveLibraryWorkflow } from 'features/workflowLibrary/hooks/useSaveWorkflow';
+import type { MouseEventHandler } from 'react';
+import { memo, useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { PiFloppyDiskBold } from 'react-icons/pi';
+
+const SaveWorkflowButton = () => {
+  const { t } = useTranslation();
+  const { onOpen } = useSaveWorkflowAsDialog();
+  const { saveWorkflow } = useSaveLibraryWorkflow();
+
+  const handleClickSave = useCallback<MouseEventHandler<HTMLButtonElement>>(
```

Comment:
If you click the save button in view mode, the dialog doesn't show. Then if you click edit, it pops up. Looks like the dialog component is only rendered in the edit view